### DISCUSSION TITLE
I have a `SyntaxError: Unexpected token 'export'` in nextjs with useFiefUserinfo

### DISCUSSION THREAD

**Discussion by michaldev** [fief-dev/fief#406] [General]
Hi. I use nextjs (14.2.3) and i configured nextjs for working with pages router - especially for fief. In general manual from fief docs works good. I have access token and working routing checking, but i have a problem with hook "useFiefUserinfo()". When I use it I got an error: 
`SyntaxError: Unexpected token 'export'`

Just this code create the problem:
`const userinfo = useFiefUserinfo();`

If I comment this line, my app works very well. Other hooks like "useFiefAccessTokenInfo" works good.

**Comment by frankie567** [comment]
Unfortunately, there are compatibility issues with newer versions of Next.js, even with pages router. I'm currently working on a new version of the JS SDK working with latest Next.js. Documentation is missing, but you can try to install it: https://www.npmjs.com/package/@fief/fief/v/0.15.0-beta.2
